Jerry_Hung.cfdg

```diff
@@ -0,0 +1,19 @@
+startshape DOCUMENT
```

Comment:
Your coding is nice and neat. I just have one question: why do you have two rules with identical name? How will you make sure that the computer randomly select the desired rule?